apps/worker/src/app/workflow/usecases/send-message/send-message-in-app.usecase.ts

```diff
@@ -156,6 +156,17 @@ export class SendMessageInApp extends SendMessageBase {
 
         inAppChannel.template.cta.action.buttons = ctaButtons;
       }
+
+      if (actor && actor.data) {
+        actor.data = await this.compileInAppTemplate(
+          actor.data,
+          command.payload,
+          subscriber,
+          command,
+          organization,
+          tenant
+        );
+      }
```

Comment:
@jainpawan21 this implementation will make that if user sets the avatar will always be shown but there could be cases where users don't want that to happen. Is this the expected behaviour?

Reply:
I will update this